diff --git a/src/racer/util.rs b/src/racer/util.rs
index ce69e4d..7f7a4b0 100644
--- a/src/racer/util.rs
+++ b/src/racer/util.rs
@@ -516,7 +516,7 @@ pub fn get_rust_src_path() -> Result<path::PathBuf, RustSrcPathError> {
 
     debug!("Nope. Trying default paths: /usr/local/src/rust/src and /usr/src/rust/src");
 
-    let default_paths = ["/usr/local/src/rust/src", "/usr/src/rust/src"];
+    let default_paths = ["@rustcSrc@", "/usr/local/src/rust/src", "/usr/src/rust/src"];
 
     for path in &default_paths {
         if let Ok(path) = validate_rust_src_path(path::PathBuf::from(path)) {
